### ISSUE TITLE
Append characters in prefix (exactly 3 char) 

### ISSUE THREAD

**Issue by Vijay-Yosi** [splitwise/TokenAutoComplete#242]
I'm using your code in my application(multiple tag like gmail to address), working fine. But while i type exactly 3 character for searching and values are shown in dropdown, select one value then  I start to type another email search in same filed means the before searching characters are append with newly searching character.

```
Eg:- I have search email "moh" select "mohan@mail.com" then start type to another mail search start with "e" means it's showing with previous searching character like "mohe"... how can i solve this. I have download this latest code and check it. This also having same issue. Please update your feedback about this issue. Thanks.
```

**Comment by mgod** [comment]
I'm not able to reproduce this issue with the sample project in emulator on my machine. This may be a device or OS version specific issue or even a keyboard problem. Can you see if you can reproduce this with the sample project and give me details about what device you're running this on?

**Comment by Vijay-Yosi** [comment]
Can you please check with version 6.0.1 marshmallow

**Comment by Mohan3** [comment]
I have check this issue in Nexus 9 device OS version 6.0.1(Marshmallow). Me too get this same issue.

**Comment by dungbk811** [comment]
I have check this issue in HTC M8 device OS version 4.4. Me too get this same issue.

**Comment by mgod** [comment]
default keyboard?

**Comment by dungbk811** [comment]
Not default, custom keyboard. I just check with default that it's right.

**Comment by sgopi93** [comment]
same issue in all os, for default keyboard also

**Comment by akshitc** [comment]
I am also getting this issue on Nexus 6, Android version 7.0 (Nougat). This issue is reproducible even in sample project as well on this device. It's like as mentioned by Vijay-Yosi, when you type 3 characters and select a token, then when you begin to type next character, the previous 3 characters got prepended before the current new character.
Also, I am not able to reproduce this issue on Moto X, Android 6.0. Only happening in Android 7.0 it seems.

**Comment by VigneshRamakrishnan** [comment]
I am also getting this issue in Nexus 6 Android 6.0.1 . The exact same issue. It is not reproducible on Android 4.x and 7.0 though.

**Comment by rpjuvvadi** [comment]
+1
Same issue. Running on Redmi 1s with AOSP Keyboard & MM 6.0.1

**Comment by mattatsyn** [comment]
+1
Issue easily reproducible on Nexus 5X hardware running 7.0 (API 24).

**Comment by duckm8795** [comment]
How can i temporarily fix this bug by edit the source code of library ????

**Comment by GAumala** [comment]
I'm having this issue with the simeji japanese keyboard

**Comment by mgod** [comment]
Closed in #333.
